examples/energy-management-app/esp32/main/include/CHIPProjectConfig.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2023 Project CHIP Authors
+ *    Copyright (c) 2024 Project CHIP Authors
```

Comment:
Why?

Reply:
I thought I had to change the date to 2024 as it only existed since 4th Jan 2024?

Reply:
I have changed these 2023-2024 as you suggested below